File: frontend/lib/models/monitoring_log.dart
class MonitoringLog {
  int id;
  String type;
  String content;

  String createdAt;
  String updatedAt;

  MonitoringLog({
    required this.id,
    required this.type,
    required this.content,
    required this.createdAt,
    required this.updatedAt,
  });

  MonitoringLog.fromJson(Map<String, dynamic> json)
      : id = json['id'] as int,
        type = json['type'] as String,
        content = json['content'] as String,
        createdAt = json['created_at'] as String,
        updatedAt = json['updated_at'] as String;

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'type': type,
      'content': content,
      'created_at': createdAt,
      'updated_at': updatedAt
    };
  }
}

File: backend/models/monitoring-log.go
package models

import (
	"time"
)

type MonitoringLog struct {
	ID        int       `json:"id" gorm:"primaryKey;autoIncrement:true"`
	Type      string    `json:"type" gorm:"not null"`
	Content   string    `json:"content" gorm:"not null"`
	CreatedAt time.Time `json:"created_at"`
	UpdatedAt time.Time `json:"updated_at"`
}
